### PR TITLE
Implement unidirectional edges API

### PR DESCRIPTION
**Pull request by christianjgreen**
This PR is an initial attempt for implement the bindings for the unidirectional edges api. 

- Would it be okay to bump the column width a bit for the C files?

### CODE REVIEW

**Comment by vihu** [comment]
I think it would be prudent to split the `h3_SUITE` at this point. Maybe have an `h3_edge_SUITE`, containing just the edge function related test cases. Thoughts?

**Comment by christianjgreen** [comment]
I agree it would be best to break this out into a separate suite. Not sure if this is an issue or not, but I noticed that the `ParisIndex` is mapped to `Sunnyvale` in the current tests.

File: c_src/h3.c [hunk 1]
```diff
@@ -690,7 +798,27 @@ static ErlNifFunc nif_funcs[] =
      {"max_k_ring_size", 1, erl_max_k_ring_size, 0},
      {"indices_are_neighbors", 2, erl_indices_are_neighbors, 0},
      {"get_unidirectional_edge", 2, erl_get_unidirectional_edge, 0},
-     {"grid_distance", 2, erl_grid_distance, 0}};
+     {"grid_distance", 2, erl_grid_distance, 0},
+     {"get_origin_from_unidirectional_edge",
+      1,
```
Comment (madninja) [inline c_src/h3.c:803]: Could you put each of these on one line, like the others?

**Review by madninja — CHANGES_REQUESTED** [review]
Looks good... One re-format request in there.

If you want to fix that test that’s in the wrong location, in another PR?

**Comment by madninja** [comment]
@christianjgreen On the 80 column question, you mean for clang-format?

File: c_src/h3.c (hunk 1 above)
Comment (christianjgreen) [reply c_src/h3.c:803]: > On the 80 column question, you mean for clang-format?

This is due to the c-lang formatting here :(

I will be following up with a PR to split some tests as well.

**Pull request closed without merging**